stainless_extraction/src/std_items.rs

```diff
@@ -49,31 +51,25 @@ impl CrateItem {
       SubsetOfFn => "stainless::Set::<T>::is_subset_of",
       SetEmptyFn => "stainless::Set::<T>::empty",
       SetSingletonFn => "stainless::Set::<T>::singleton",
+      BoxNew => "std::boxed::Box::<T>::new",
     }
   }
 
   /// An item can (unfortunately) be in a different crate than the first part
   /// of its [path()] suggests. This is due to the aliasing of `alloc` and other
   /// layers under `std`.
   pub fn crate_name(&self) -> &'static str {
-    self.path().splitn(2, "::").next().unwrap()
+    match self {
+      BoxNew => "alloc",
```

Comment:
This is fine for now but we should start thinking about how to better do this at scale.

Reply:
Indeed, this is tracked by #48.